### DISCUSSION TITLE
Allow create/update api calls to be dry runs

### DISCUSSION THREAD

**Discussion by moekify** [directus/directus#8647] [Feature Requests]
For testing purposes or i.e. running stuff in a development environment and testing if an api call works, it would be neat to have the ability to dry run api calls. 

They could return either a similar object with some additional param, or just a generic, yeah this would be okay. 

Can elaborate a bit more later, but wanted to jot this down before I forget :)

**Comment by rijkvanzanten** [comment]
Interesting! I'm wondering how we can effectively do that in SQL queries and with all the side effects in mind 🤔 

For example, for things like auto-incremented IDs, or other "database magic", we'll have to figure out how to do it without committing. Maybe we can cancel out the transaction on the SQL level to still get the results, but then not commit the trx? 🤔

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
